packages/teleport/src/Invite/useInvite.ts

```diff
@@ -58,6 +58,7 @@ export default function useInvite(tokenId: string) {
     auth2faType,
     fetchAttempt: fetchAttempt.attempt,
     submitAttempt: submitAttempt.attempt,
+    setSubmitAttempt: submitAttempt.setAttempt,
```

Comment:
instead of this, lets define a `clearSubmitAttempt`, that `setAttempt({status: '', statusText: ''})